Jenkinsfile

```diff
@@ -60,6 +60,22 @@ pipeline {
         }
       }
     }
+
+    stage ('Building and Pushing Gems') {
+      environment {
+        GEM_HOST_API_KEY = credentials('jenkins-rubygems-api-key')
+      }
+      when {
+        buildingTag()
+      }
+      steps {
+        container('builder') {
+          script { 
+            sh 'make release'
+          }
+        }
+      }
+    }
```

Comment:
Let's just move this before the docker images. We want to keep an invariant saying "if the docker image exists, then the ruby gem has been published".